### ISSUE TITLE
Missing styles in build vs start

### ISSUE THREAD

**Issue by loliver** [seek-oss/playroom#29]
Hi there,

I'm having all sorts of issues that I can't seem to pin down with my built assets not retaining a portion of their styles, using styled-components.

Eg:
<img width="338" alt="screen shot 2018-11-29 at 4 34 21 pm" src="https://user-images.githubusercontent.com/3507971/49201403-e4250000-f3f4-11e8-9a85-ee5f70a5c66e.png">

vs

<img width="335" alt="screen shot 2018-11-29 at 4 35 45 pm" src="https://user-images.githubusercontent.com/3507971/49201408-e5562d00-f3f4-11e8-925b-689f5845eee8.png">

My webpack config as is follows:
```JavaScript
  webpackConfig: () => ({
    module: {
      rules: [
        {
          test: /\.js$/,
          loader: 'babel-loader',
          query: {
            plugins: ['lodash'],
            presets: [
              '@babel/preset-react',
              [
                '@babel/preset-env',
                {
                  targets: ['last 2 versions', 'ie >= 11']
                }
              ]
            ]
          },
          exclude: [
            /node_modules\/(?!(@anz)\/).*/
          ]
        },
        {
          test: /\.(otf|ttf|eot|woff(2)?)(\?[a-z0-9=&.]+)?$/,
          loader: 'file-loader'
        },
        {
          test: /\.(jpe?g|png|gif|svg|ico)$/,
          loader: 'file-loader?name=assets/[name].[ext]',
          exclude: '/node_modules/'
        }
      ]
    },
    resolve: {
      extensions: ['.js', '.jsx', '.json']
    }
  })
```

and frame component is:

```JavaScript
import React from 'react'
import { StyleSheetManager } from 'styled-components'

export default ({ children, frameWindow }) => (
  <StyleSheetManager target={frameWindow.document.head}>
    {children}
  </StyleSheetManager>
)
```

Everything works perfectly using `start` but the built assets seem to lose some of their font-family, padding, and margin, while color and border seem to stay attached somehow. No webpack errors in the terminal while building, and the generated style tags in each frame's head are empty in `build`, but full of expected classes etc in `start`.

**Comment by markdalgleish** [comment]
Any chance I could get a minimal reproduction of this issue somewhere?

**Comment by loliver** [comment]
I'll find somewhere to put the dist out on the internet for the time being, will let you know when/where it is once I figure that out.

**Comment by markdalgleish** [comment]
Source is more important, so I can have a poke around at the build.

**Comment by loliver** [comment]
https://github.com/patrickmarabeas/nothing-to-see-here/tree/stuff

Pat has kindly donated a private repo so it's less likely I'll get fired out of a cannon in to the sun for doing this, haha.

**Comment by markdalgleish** [comment]
Thanks for letting me see your usage—the reproduction is a bit heavy, though, so I don't really have the time to dig in. Are you able to create a new project and reproduce the issue there? It'll probably help you narrow down exactly what's wrong, too.

**Comment by loliver** [comment]
Yeah no probs, I'll have a look in to it tomorrow and see what I can pull out.

**Comment by loliver** [comment]
https://github.com/patrickmarabeas/nothing-to-see-here

Hopefully this is what you were looking for, it's only got our Text component which depends on global styles, injectGlobal doesn't appear to be applied which may be the issue here but not sure how to fix that with the `StyleSheetManager` component.

**Comment by markdalgleish** [comment]
Is this an issue with styled-components?

**Comment by loliver** [comment]
I think it could be, tried out manually running `injectGlobal` in `frame-component` as per https://stackoverflow.com/questions/45489383/how-to-inject-global-styles-while-rendering-server-side-with-styled-components?rq=1 but that didn't seem to do the trick, will keep digging.
